doc/static-route/SONiC_static_route_bfd_hld.md

```diff
@@ -0,0 +1,300 @@
+# Scope
+
+This document describes the design of using BFD to detect static route nexthop and based on BFD session state, installing, updating, and removing static route to the system.<br  /> 
+
+One important assumption is that the BFD session (for a static route nexthop) in this BFD Static Route application is not shared with any other applications (i.e., creating or deleting static route BFD session has no impact on other applications).<br /> 
```

Comment:
Any potential applications (other than StaticRouteBfd) that can listen BFD_SESSION_TABLE information from STATE_DB and take any action?